modal.js

```diff
@@ -14,8 +13,7 @@ const data = [
     html: 'html',
     css: 'css',
     javascript: 'javasctipt',
```

Comment:
- [ ] [OPTIONAL] This code works right now although you can summarize this into a `technologies` array that has all the technologies. that will make this code scalable in case two projects have more than just these technologies and also if you have different projects having different numbers of technologies used (which is a realist scenario)